### DISCUSSION TITLE
您好，购买的专业版完全无法使用，收不到验证码，请帮忙协助退款，购买凭证如下：

### DISCUSSION THREAD

**Discussion by wwei20240912** [getlantern/lantern-forum-cn#945] [General]
您好，购买的专业版完全无法使用，收不到验证码，请帮忙协助退款，购买凭证如下：
![Uploading 蓝灯付款26272_.pic.jpg…]()

**Comment by wwei20240912** [comment]
@Mike777Khoo 您好，购买的专业版完全无法使用，收不到验证码，请帮忙协助退款，

**Comment by MaxMKMax** [comment]
@wwei20240912 您好. 非常抱歉啊! 我帮您和技术人员拿验证码. 请您别退款. 我这里尽全力帮您弄好. 谢谢!

**Comment by MaxMKMax** [comment]
@wwei20240912 请您发您的账号邮箱和支付单据到我的邮箱. 然后记得告诉我您的是哪一个邮箱.
mike@getlantern.org
